doc/source/security/encrypting-threads.rst

```diff
@@ -0,0 +1,96 @@
+.. _encrypting-threads:
+
+================================================================================
+Working with Background Encryption Threads
+================================================================================
+
+:Availabiliity: This feature is **Experimental**.
+
+Encryption threads in the background allow you to perform some encryption and
+decryption tasks in real-time. 
+
+You would use encryption threads for the following purposes:
+
+* Encryption threads can encrypt existing tablespaces. Encryption
+  threads allow encryption to be applied to all or some of the existing
+  tablespaces, you can exclude tablespaces from rotation, in a background process. You
+  can encrypt existing tables with the Master key, but you must do this
```

Comment:
here also existing tables/existing tablespaces

Reply:
changed tables to tablespaces in the last line.